client/state/data-layer/wpcom-api-middleware.js

```diff
@@ -17,16 +17,16 @@ const mergedHandlers = mergeHandlers(
 const shouldNext = action => {
 	const meta = action.meta;
 	if ( ! meta ) {
-		return true;
+		return false;
 	}
 
 	const data = meta.dataLayer;
 	if ( ! data ) {
-		return true;
+		return false;
```

Comment:
I think we should leave these here because we should definitely pass an action through to the next middleware if there is no `meta` on it. we need to evaluate the tests if they are wrong.